e2e/cypress/integration/at_mentions/at_mentions_spec.js

```diff
@@ -86,6 +89,14 @@ let townsquareChannelId;
 let offTopicChannelId;
 
 describe('at-mention', () => {
+    function ignoreUncaughtException() {
+        cy.on('uncaught:exception', (err) => {
+            expect(err.message).to.include('notification.close is not a function');
+
+            return false;
+        });
```

Comment:
Ignore notification.close is not a function on this spec. Spying on Notification randomly raised uncaught exception error.